docs/REST API/rest-api.md

```diff
@@ -9193,7 +9295,7 @@ Install software (package or App Store app) on a macOS, iOS, iPadOS, Windows, or
 
 _Available in Fleet Premium._
 
-`GET /api/v1/fleet/software/install/results/:install_uuid`
+`GET /api/v1/fleet/software/:install_uuid/install/results`
```

Comment:
I missed this one in my first review, needs to be updated to match the example:
```suggestion
`GET /api/v1/fleet/software/install/:install_uuid/results`
```

Reply:
thank you!